### PR TITLE
Fixes type error in range()

### PR DESCRIPTION
**Pull request by DefCon-007**
@jdf  The for loop in line 31 uses range and it gives type error since `fillGap` is a float. The complete error is as follows : 

`processing.app.SketchException: TypeError: range() integer end argument expected, got float.`

This PR fixes it.

### CODE REVIEW

File: examples.py/Basics/Color/WaveGradient.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 """
 
 amplitude = 30
-fillGap = 2.5
+fillGap = 2
```
Comment (jdf) [inline examples.py/Basics/Color/WaveGradient.py]: Leave this 2.5

Comment (jdf) [inline examples.py/Basics/Color/WaveGradient.py]: And then 

```python
import math

# ...

for filler in range(int(math.ceil(fillGap))):`
```

**Comment by DefCon-007** [comment]
@jdf I have made the suggested changes please review

**Comment by jdf** [comment]
Thank you.

**Pull request merged**